experimental/uefi/runtime/src/lib.rs

```diff
@@ -17,6 +17,9 @@
 #![no_std]
 #![feature(never_type)]
 
+#[cfg(test)]
+extern crate std;
+
```

Comment:
Could you add a comment why this was added? :)

Reply:
Turns out it isn't; I think it's a leftover from some intermediate state when I tried to get tests to work.